strawberryfields/gbs/similarity.py

```diff
@@ -52,9 +101,6 @@
 def sample_to_orbit(sample: list) -> list:
     """Provides the orbit corresponding to a given sample.
 
-    Orbits are simply a sorting of integer photon number samples in non-increasing order with the
-    zeros at the end removed.
-

```

Comment:
I found these lined unnecessary, especially now that we have defined properly the orbits and events in the module level docstrings.